### PR TITLE
bug fixes in get_nearest; modified logic for combined get_nearest

### PR DESCRIPTION
**Pull request by shivam05011996**
Related to #36 #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,8 +22,8 @@ word vector space data. Here are two ways to get that.
 #### Download pre-computed sample data
 
 ```bash
-$ wget https://s3.amazonaws.com/deepcompute-public-data/wordvecspace/small_test_data.tgz
-$ tar zxvf small_test_data.tgz
+$ wget https://s3.amazonaws.com/deepcompute-public-data/wordvecspace/test_data-0_5_4.tgz
+$ tar test_data-0_5_4.tgz
```
Comment (RamanjaneyuluIdavalapati) [inline README.md]: It should be tar zxvf test_data-0_5_4.tgz

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
 # WordVecSpace
 A high performance pure python module that helps in loading and performing operations on word vector spaces created using Google's Word2vec tool.
 
-This module has ability to the load data into memory using `WordVecSpaceMem` and it can also support performing operations on the data which is on the disk using `WordVecSpaceAnnoy` and `WordVecSpaceDisk`.
+This module has ability to the load data into memory using `WordVecSpaceMem` and it can also support performing operations on the data which is on the disk using `WordVecSpaceAnnoy` and                   `WordVecSpaceDisk`.
 
 ## Installation
-> Prerequisites: >=Python3.5.2
+> Prerequisites: Python3.5.2
```
Comment (RamanjaneyuluIdavalapati) [inline README.md]: Why not Prerequisites: >=Python3.5.2,  so that people will understand that it will work for above python3.5.2

File: README.md [hunk 3]
```diff
@@ -41,7 +42,7 @@ $ git clone https://github.com/tmikolov/word2vec.git
 
 # 1. Navigate to the folder word2vec
 # 2. open demo-word.sh for editing
-# 3. Edit the command "time ./word2vec -train text8 -output vectors.bin -cbow 1 -size 200 -window 8 -negative 25 -hs 0 -sample 1e-4 -threads 20 -binary 1 -iter 15" ----to----> "time ./word2vec -train text8 -output vectors.bin -cbow 1 -size 5 -window 8 -negative 25 -hs 0 -sample 1e-4 -threads 20 -binary 1 -save-vocab vocab.txt -iter 15" to get vocab.txt file also as output.
+# 3. Edit the command "time ./word2vec -train text8 -output vectors.bin -cbow 1 -size 200 -window 8 -negative 25 -hs 0 -sample 1e-4 -threads 20 -binary 1 -iter 15" ----to----> "time ./word2vec -train     text8 -output vectors.bin -cbow 1 -size 5 -window 8 -negative 25 -hs 0 -sample 1e-4 -threads 20 -binary 1 -save-vocab vocab.txt -iter 15" to get vocab.txt file also as output.
```
Comment (RamanjaneyuluIdavalapati) [inline README.md]: Extra space

File: README.md [hunk 4]
```diff
@@ -96,7 +97,7 @@ $ wordvecspace convert /home/user/bindata /home/user/output_dir
 
 `WordVecSpaceMem` and `WordVecSpaceDisk` is a bruteforce algorithm which compares given word with all the words in the vector space
 
-`WordVecSpaceAnnoy` takes wordvecspace output_dir as input and creates annoy indexes in another file (index file). Using this file `annoy` gives approximate results quickly. For better understanding of `Annoy` please go through this [link](https://github.com/spotify/annoy)
+`WordVecSpaceAnnoy` takes wordvecspace output_dir as input and creates annoy indexes in another file (index file). Using this file `annoy` gives approximate results quickly. For better understanding of   `Annoy` please go through this [link](https://github.com/spotify/annoy)
```
Comment (RamanjaneyuluIdavalapati) [inline README.md]: Extra space

File: README.md (hunk 2 above)
Comment (shivam05011996) [reply README.md]: it does not work for python 3.6

File: README.md (hunk 3 above)
Comment (shivam05011996) [reply README.md]: will remove

File: .travis.yml [hunk 5]
```diff
@@ -2,10 +2,11 @@ language: python
 python:
 - '3.5'
 before_install:
-- wget 'https://s3.amazonaws.com/deepcompute-public-data/wordvecspace/small_test_data.tgz'
-  && tar xvzf small_test_data.tgz
-- export WORDVECSPACE_DATADIR='small_test_data'
+    - wget 'https://s3.amazonaws.com/deepcompute-public-data/wordvecspace/test_data-0_5_4.tgz'
```
Comment (RamanjaneyuluIdavalapati) [inline .travis.yml]: Extra space

**Pull request merged**